IdentityCore/src/MSIDDeviceId.m

```diff
@@ -198,4 +199,13 @@ + (void)setIdValue:(NSString *)value
     [(NSMutableDictionary *)[self deviceId] setObject:value forKey:key];
 }
 
++ (NSString *)deviceHardwareType
+{
+    struct utsname sysinfo;
+    int retVal = uname(&sysinfo);
```

Comment:
I assume this is public API? (just double checking, because I'm not familiar with it)

Reply:
yes it is an API in utsname.h file in sys module

![image](https://github.com/AzureAD/microsoft-authentication-library-common-for-objc/assets/9446116/8d591b82-a3ea-479f-9dfd-5a253b2ffabb)